manifests/set.pp

```diff
@@ -166,12 +166,13 @@
 
     # sync if needed by helper script
     exec { "sync_ipset_${title}":
-      path    => ['/sbin', '/usr/sbin', '/bin', '/usr/bin', '/usr/local/bin', '/usr/local/sbin'],
+      path        => ['/sbin', '/usr/sbin', '/bin', '/usr/bin', '/usr/local/bin', '/usr/local/sbin'],
       # use helper script to do the sync
-      command => "ipset_sync -c '${config_path}'    -i ${title}${ignore_contents_opt}",
+      command     => "ipset_sync -c '${config_path}'    -i ${title}${ignore_contents_opt}",
       # only when difference with in-kernel set is detected
-      unless  => "ipset_sync -c '${config_path}' -d -i ${title}${ignore_contents_opt}",
-      require => Package['ipset'],
+      unless      => "ipset_sync -c '${config_path}' -d -i ${title}${ignore_contents_opt}",
```

Comment:
Why is the unless test not working on EL8?

Reply:
Looked into this briefly, but came up empty. So for now all I know is it doesn't work, but not why :/

Reply:
@bastelfreak Heya, I know this is a very stupid solution and probably not a proper fix. But I can't seem to find another solution. But it would be nice to finally have these eye soar ipset sync messages gone from our foreman reports :)

Reply:
took me a bit to read the code, but I think the refreshonly is fine. We notify the exec resource when keep_in_sync is true, and it doesn't make sense to always run it.